eng/Version.Details.xml

```diff
@@ -290,6 +290,11 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>7d981168a2a2ed7d2190776a08ece88665cba727</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.Web.Xdt" Version="7.0.0-preview.22419.1">
```

Comment:
I'm surprised by this addition. Why not just take the latest version in the `dotnet-public` feed❔ That matches what's available on NuGet.org.

Reply:
This is needed for source-build. dotnet/aspnetcore consumes dotnet/xdt, and this is the correct location for the dependency.

Reply:
Why❔ I thought source-build builds just overrode what's used in `msbuild` and Version.Details.xml is ignored except by Maestro++ i.e. for automated dependency PRs.

In any case, you won't get this to build w/o fixing Versions.props. But I'd still rather _not_ add a new subscription unless absolutely required.

Reply:
There is a source-build tag: ` <SourceBuild RepoName="xdt" ManagedOnly="true" />`

Source-build consumes Versions.Details.xml to determine correct SHA of all dependent repos.

No fix is needed in version.props for this to work.

The failing source-build leg is due to a different issue - XDT repo is not publishing anything to public feeds and source-build is missing the transport package. Working on those changes now - will update the PR later.

Reply:
> No fix is needed in version.props for this to work.

Update in that file must place `MicrosoftWebXdtVersion` in the `Label="Automated"` property group or builds will continue to fail.

Reply:
Thanks! This is interesting - local build did not hit that validation.